README.md

```diff
@@ -50,49 +50,64 @@ Sometimes it can be useful to override these options without changing or recompi
 
 Using Maven:
 
-    mvn -Dcucumber.options="..." test
+    mvn -Dcucumber.features="..." -Dcucumber.glue="..." test
 
 Using Gradle:
 
-    gradlew -Dcucumber.options="..." test
-
-Let's look at some things you can do with `cucumber.options`. Try this:
-
-    -Dcucumber.options="--help"
-
-That should list all the available options.
+    gradlew -Dcucumber.features="..." -Dcucumber.glue="..." test
 
 *IMPORTANT*
 
-When you override options with `-Dcucumber.options`, you will completely override whatever options are hard-coded in
+When you override options with `-Dcucumber.<option>`, you will completely override whatever options are hard-coded in
```

Comment:
This is no longer true. CLI arguments take precedence over all arguments. 

https://github.com/cucumber/cucumber-jvm/tree/master/core#properties-environment-variables-system-options

Reply:
So what's wrong with the docs? `-Dcucumber.<option>` means `-Dcucumber.features`... etc. So `-Dcucumber....` from CLI should override all arguments?

Reply:
The bit that is wrong is "you will completely override whatever options are hard-coded in your `@CucumberOptions` or in the script calling `cucumber.api.cli.Main`."

CLI options override the system properties now.

Reply:
So I think we should remove this overriding rules, along with the properties table below, and let the user consults the info from `--help`, do you agree?

Reply:
I also resolve this conversation since it's related to the first one